pandas/tests/io/parser/test_common.py

```diff
@@ -2127,6 +2127,12 @@ def test_first_row_bom(all_parsers):
     expected = DataFrame(columns=["Head1", "Head2", "Head3"])
     tm.assert_frame_equal(result, expected)
 
+    # see gh-36343
```

Comment:
can you make this a new test

Reply:
done, I think test failures are unrelated. https://dev.azure.com/pandas-dev/pandas/_build/results?buildId=42772&view=ms.vss-test-web.build-test-results-tab&runId=2173848&resultId=150762&paneView=debug